frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideInNewBranchBelowAction.java

```diff
@@ -81,21 +81,38 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       return;
     }
 
-    var branchName = createOrCheckoutNewBranch(project, selectedVcsRepository.get(), gitMacheteParentBranch.get().getName(),
+    var parentName = gitMacheteParentBranch.get().getName();
+    var branchName = createOrCheckoutNewBranch(project, selectedVcsRepository.get(), parentName,
         GitMacheteBundle.message("action.GitMachete.BaseSlideInBranchBelowAction.dialog.title"));
     if (branchName == null) {
       log().debug("Name of branch to slide in is null: most likely the action has been canceled from dialog");
       return;
     }
 
+    var entryAlreadyExistsBelowGivenParent = branchLayout.get().findEntryByName(parentName)
```

Comment:
`branchLayout.get()` appears a couple times, maybe change `branchLayout`'s declaration to `var branchLayout = getBranchLayout(anActionEvent).getOrNull()`.

Reply:
Also, pls add a TODO for #430 here